src/GovNotifyHandler.ts

```diff
@@ -46,7 +46,8 @@ class GovNotifyHandler implements LambdaInterface {
 						throw err;
 					}
 				}
-				await SendEmailProcessor.getInstance(logger, metrics, GOVUKNOTIFY_API_KEY, this.environmentVariables.sessionEventsTable()).processRequest(body);
+				const govnotifyServiceId = GOVUKNOTIFY_API_KEY.substring(GOVUKNOTIFY_API_KEY.length - 73, GOVUKNOTIFY_API_KEY.length - 37);
+				await SendEmailProcessor.getInstance(logger, metrics, GOVUKNOTIFY_API_KEY, govnotifyServiceId, this.environmentVariables.sessionEventsTable()).processRequest(body);
```

Comment:
could we wrap it in a try/catch pls incase substring throws an exception and call SendEmailProcessor only if  govnotifyServiceId is truthy

Reply:
Done